pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -52,7 +52,12 @@ buildPythonApplication rec {
   ];
 
   nativeBuildInputs = [
-    pkg-config sphinx ncurses
+    pkg-config ncurses
+    sphinx
+    furo
+    sphinx-copybutton
+    sphinxext-opengraph
+    sphinx-inline-tabs
     installShellFiles
```

Comment:
I'd suggest one package per line only and sorting

Reply:
I have now sorted `nativeBuildInputs` somewhat but not the last five packages because I like to keep them in the original order provided by upstream: https://github.com/kovidgoyal/kitty/blob/f67995d5d35acd5db4a9086997a93b82f21b2c6a/docs/requirements.txt